geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+
+# Bond lengths: rm^(1)     (angstroms)
+# Bond angles:  thetam^(1) (degrees)
+
+F  Si   1.5882(9)
+Si H    1.4691(1)
+F  Si H 108.304(7)
+
+# Bond lengths: re^se     (angstroms)
+# Bond angles:  thetae^se (degrees)
+
+F  Si   1.59048(6)
+Si H    1.46948(9)
+F  Si H 108.304(9)
+
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+
+Si D    1.4755(29)
+Si F    1.5931(26)
+D  Si D 110.472(324)
+D  Si F 108.452(561)
+
+# Bond lengths: re     (angstroms)
+# Bond angles:  thetae (degrees)
+
+Si H    1.46644(447)
+Si F    1.59146(237)
+H  Si H 110.805(506)
+H  Si F 108.102(292)
+
+NAME = F3N 2015
+# Point Group: C3v
+# Shape: Trigonal pyramidal
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Najib H (2015) Experimental values of the rotational and vibrational constants and equilibrium structure of
+nitrogen trifluoride. J Mol Spectrosc 312:1-5
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+N F   1.37132(66)
+F N F 102.11761(46)
+
+# Bond lengths: re     (angstroms)
+# Bond angles:  thetae (degrees)
+
+N F   1.36757(58)
+F N F 101.8513(10)
+
+NAME = F3P 2014
+# Point Group: C3v
+# Shape: Trigonal pyramidal
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Najib H (2014) Experimental rovibrational constants and equilibrium structure of phosphorus trifluoride. J Mol
+Spectrosc 305:17-21
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+P F   1.56324405(11)
+F P F 97.752232(29)
+
+# Bond lengths: re     (angstroms)
+# Bond angles:  thetae (degrees)
+
+N F   1.560986(43)
+F P F 97.566657(64)
+
+NAME = F4Te
+# Point Group: C2v
+# Shape: Seesaw
+# Bond lengths: rg     (angstroms)
+# Bond angles:  thetag (degrees)
+# Source Paper: Shlykov SA, Giricheva NI, Titov AV, Szwak M, Lentz D, Girichev GV (2010) The structures of tellurium(IV)
+halides in the gas phase and as solvated molecules. Dalton Trans 39(13):3245-3255
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Te    F(ax)    1.899(4)
+Te    F(eq)    1.846(4)
+F(ax) Te F(ax) 164.3(12)
+F(eq) Te F(eq) 99.5(3)
+
+NAME = H2I2 2009
+# Point Group: C2h
+# Shape: Linear --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Coudert LH, Belov SP, Willaert F, McElmurry BA, Bevan JW, Hougen JT (2009) Submillimeter spectrum and
+analysis of vibrational and hyperfine coupling effects in (HI)2. Chem Phys Lett 482(4-6):180-188
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+HI HI    4.56372(1)
+H  I...I 46.405(1)
+
+NAME = H2I2Si
+# Point Group: C2v
+# Shape: Seesaw
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Arsenault EA, Obenchain DA, Orellana W, Novick SE (2017) Nuclear quadrupole coupling in SiH2I2 due to the
+presence of two iodine nuclei. J Mol Spectrosc 338(2):72-76
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Si I    2.4236(19)
+Si H    1.475(21)
+I  Si I 111.27(13)
+H  Si H 105.9(19)
+
+NAME = H2O2Si
```

Comment:
How is this (Dioxasilacyclopropane) distinguished from the other H2O2Si in the book (Dihydroxysilylene)? I had been putting years next to the chemical formulas in my entries. That is not a perfect solution, so I'm open to suggestions.

Reply:
Hmmm we do need a way to deal with structural isomers. I guess one is Cs and the other is C2v. I would need to think about the case where two isomers have the same geometry.

Reply:
Good point. If a user were to ask for the z-matrix for H2O2Si without specifying the point group, then the program can say: "Warning, the database has two entries for that molecule, one with point group C2v and one with point group Cs. Which one do you want?". Alternatively, if the user asks for the C2v molecule, then they will get the C2v one.

Reply:
Sure and if we have two of the same geometry then the user can choose once again.